projects/remote_transceiver/inc/remote_transceiver.h

```diff
@@ -157,7 +157,7 @@ class Listener : public std::enable_shared_from_this<Listener>
      * @param acceptor tcp::acceptor configured with desired host and port
      * @param db       SailbotDB instance - the Listener requires that it takes ownership of the db
      */
-    Listener(bio::io_context & io, tcp::acceptor acceptor, SailbotDB && db);
+    Listener(tcp::acceptor acceptor, SailbotDB && db, bio::strand<bio::io_context::executor_type> strand);
```

Comment:
I think it would be cleaner to just do what's done in the [official example](https://github.com/boostorg/beast/blob/f8ff44d072803e8c193946818ee7d239c5834b10/example/http/server/async/http_server_async.cpp#L336). Meaning instead of taking an acceptor and strand as arguments, take an io_context and tcp::endpoint{address, port} and make a new strand and acceptor in the constructor.

Reply:
modified to follow example and constructor now just takes an endpoint